apps/contourclock/app.js

```diff
@@ -29,14 +29,17 @@
   }
   let showExtras = function() { //show extras for a limited time
     drawExtras();
-    if (extrasTimeout) clearTimeout(extrasTimeout);
+    //if (extrasTimeout) clearTimeout(extrasTimeout);
```

Comment:
What was the reasoning for doing this? It feels like this should really be in there...